optoforce/README.md

```diff
@@ -10,10 +10,24 @@ ROS driver for the [Optoforce sensor](http://optoforce.com/3dsensor/).
 
 ## Driver description
 
-The driver relies on the pySerial library to get data from the sensors through USB.
+The driver relies on the pySerial library to get data from the sensors through USB. The module optoforce is completely independant from ROS and can be used as such.
 
-It publishes a geometry_msgs/WrenchStamped for every sensor.
+The ROS node built with this driver publishes messages of type `geometry_msgs/WrenchStamped` for every sensor.
 
+## Installation of udev rule
+
+If you want the optoforce sensors to be named in `/dev/` using their serial number, like `/dev/optoforce_ISE174`, you'll want to do the following steps.
```

Comment:
I would possibly add more details here as why you wouls want the sensors to be named in this way, such as to remove the dependency on the order which the sensors are plugged in etc.